docs/_templates/sidebar/brand.html

```diff
@@ -11,31 +11,31 @@
     <img class="sidebar-logo only-dark" src="{{ pathto('_static/' + theme_dark_logo, 1) }}" alt="Dark Logo"/>
   </div>
   {%- endif %}
-  <div class="sd-d-flex-row">
+  {% if not theme_sidebar_hide_name %}
+  <span class="sidebar-brand-text">{{ docstitle if docstitle else project }}</span>
+  {%- endif %}
+  {% endblock brand_content %}
+</a>
+<div class="sd-d-flex-row sd-align-major-center">
   <a class="github-button" href="https://github.com/jina-ai/jina" data-icon="octicon-star" data-show-count="true" aria-label="Star jina-ai/jina on GitHub">Star</a>
-  {% if versions %}
-    <select id="versions" onChange="window.location.href=this.value"></select>
+  <select onChange="window.location.href=this.value" class="version-select">
+    {% if versions %}
```

Comment:
Let's add the dropdown only if `versions` is defined (like in the `master` branch). This is helpful if someone is building locally for a single branch using `bash makedocs.sh local-only`.